testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -50,8 +50,8 @@ const GENESIS_HELM_RELEASE_NAME: &str = "genesis";
 const APTOS_NODE_HELM_CHART_PATH: &str = "terraform/helm/aptos-node";
 const GENESIS_HELM_CHART_PATH: &str = "terraform/helm/genesis";
 
-// cleanup namespaces after 30 min unless "keep = true"
-const NAMESPACE_CLEANUP_THRESHOLD_SECS: u64 = 1800;
+// cleanup namespaces after 1 hour 30 minutes unless "keep = true"
+const NAMESPACE_CLEANUP_THRESHOLD_SECS: u64 = 5400;
```

Comment:
Can we have this tunable by CLI args? https://github.com/aptos-labs/aptos-core/blob/main/testsuite/forge-cli/src/main.rs#L129-L135 

It would be bad to keep a crashed land-blocking namespace around for 90 min, since the test itself is short